tests/test_cad_to_post_wtm.py

```diff
@@ -4,8 +4,8 @@
 This test covers the setup and solution of a three-dimensional
 turbulent fluid flow and heat transfer problem in a mixing elbow. The mixing
 elbow configuration is encountered in piping systems in power plants and
-processindustries. It is often important to predict the flow field and
-temperature field in the area of the mixing regionin order to properly design
+process industries. It is often important to predict the flow field and
+temperature field in the area of the mixing region in order to properly design
```

Comment:
```suggestion
temperature field in the area of the mixing region to properly design
```
The "in order" part is unnecessary. It just adds extra words.

Reply:
@prmukherj Yes, we can remove "in order" here.